helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -408,3 +408,54 @@ configMaps: []
   #     key: |
   #       this is the content of the configmap
   #       when mounted, key will be the name of the file
+
+# specify the fe proxy deploy or not.
+feProxy:
+  # specify the fe proxy deploy or not.
+  enabled: true
+  replicas: 1
+  image:
+    repository: nginx
+    tag: 1.24.0
+  resources:
+    limits:
+      cpu: 1
+      memory: 2Gi
+    requests:
+      cpu: 1
+      memory: 2Gi
+  # set the resolver for nginx server
+  resolver: kube-dns.kube-system.svc.cluster.local
```

Comment:
can we make this default?

Reply:
the default value is also kube-dns.kube-system.svc.cluster.local. I will set it to empty string in values.yaml